scripts/projectData.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+var projectData = [
+  {
+    name:         'Project 1',
+    image:        'img/blank.png',
```

Comment:
You might consider changing these to placeholders, either using placehold.it, or one of the other image placeholder sites. That will make it easier to see your layout clearly.